### DISCUSSION TITLE
Showing OG posts on blogs & Hiding notes number

### DISCUSSION THREAD

**Discussion by velasca30** [AprilSylph/XKit-Rewritten#320] [Suggestion Box]
Hi!

I know we can hide reblogs on the board, but is there a way to also do that on people's blogs? 

Also, it would be nice to have a feature where it hides the number of notes of others except ours, since competition and comparison is a real problem and the eye catches the number quick and messes with our mental health. Maybe a way to still click to see who commented, reblogged and liked but no number at first glance?

**Comment by marcustyphoon** [comment]
Kat from Tumblr staff seemed to imply the first feature you mentioned seems like a possibility from Tumblr's side! Not sure if that post was a "this is probably coming" or "we'll consider it," however. https://wip.tumblr.com/post/659970876190621696/is-this-a-venue-for-feature-requests-heres

(I can't speak to the possibility of having it as a feature in XKit Rewritten if it does not come to Tumblr or about your second request, April will have to comment about those!)
